index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <!--link to the css style sheets-->
+    <link rel="stylesheet" href="./assets/css/styles.css" />
+
+    <!--the title of the website-->
+    <title>Job Interview Quiz Home</title>
+
+    <!-- code to change the favicon-->
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./assets/img/question.png"
+      alt="favicon"
+    />
+
+    <meta name="author" content="Alexandru Tanase" />
+
+    <!-- website keywords and description used by search engines-->
+    <meta
+      name="keywords"
+      content="Interview, Questions, Quiz, Quick Quiz, Job Placement"
+    />
+    <meta name="description" content="Job Interview Quick Quiz Home page" />
+
+    <!--make the page responsive to the with of the display screen-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <header>
+      <!--Contains the nav bar elements-->
+      <nav class="navigation-bar">
+        <div class="navigation-bar-item">
+          <a href="./index.html">Home</a>
+        </div>
+
+        <div class="navigation-bar-item">
+          <a id="high-score" href="./highscore.html">High Scores</a>
+        </div>
+      </nav>
+    </header>
+
+    <!--the main content of the page-->
+    <main id="main">
+      <!-- <section class="timer-section">
+        <div id="timer">
+          <h2>Timer</h2>
+
+          <p>Time remaining: <span id="time-span">60</span></p>
+        </div>
+      </section>
+
+      <section class="question-section" id="question-content">
+        <div class="question-section-question">
+          <p>This is where the question is?</p>
+        </div>
+
+        <ul class="quest-section-options">
+          <li>1</li>
+          <li>2</li>
+          <li>3</li>
+          <li>4</li>
+        </ul>
+
+        <div class="question-section-decision">
+          <p>Correct! or Wrong!</p>
+        </div>
+      </section> -->
+
```

Comment:
If code is not being used, please delete it rather than just comment it out. This looks quite messy,

Reply:
Done